libvirt.go

```diff
@@ -137,14 +138,21 @@ func (d *Driver) GetURL() (string, error) {
 	return "", nil
 }
 
-func (d *Driver) getConn() (*libvirt.Connect, error) {
+func (d *Driver) getConn() (*golibvirt.Libvirt, error) {
 	if d.conn == nil {
-		conn, err := libvirt.NewConnect(connectionString)
+		c, err := net.DialTimeout("unix", "/var/run/libvirt/libvirt-sock", 2*time.Second)
```

Comment:
This probably is not working with https://libvirt.org/daemons.html#modular-sockets